### PR TITLE
Migration docs

### PR DESCRIPTION
**Pull request by sowmyaraj92**
To update the steps/flow to migrate to Postman v8.

### CODE REVIEW

File: src/pages/docs/administration/migrating-to-v7.md [hunk 1]
```diff
@@ -23,24 +27,25 @@ This section describes the following topics:
 
 ## Why migrate?
 
-If you are a team user, you must migrate to Postman v7 to leverage fine-grained role-based access of Postman’s features. If you stay back on the older version of Postman app, you will not be able to:
+If you are a team user, you must migrate to Postman v8 to leverage fine-grained role-based access of Postman’s features. If you stay back on the older version of Postman, you will not be able to:
 
 * Use role-based permissions which come with a fine-grained access management of some of Postman’s prominent features.
 
-* Access new features and updates because new feature development will be based on Postman v7.
+* Access new features and updates because new feature development will be based on Postman v8.
```
Comment (cyrielle-hada) [inline src/pages/docs/administration/migrating-to-v7.md:34]: @sowmyaraj92 can we redirect to the [release notes](https://www.postman.com/downloads/release-notes/) so users will be able to see features/bug fixes released from v8?

File: src/pages/docs/administration/migrating-to-v7.md [hunk 2]
```diff
@@ -23,24 +27,25 @@ This section describes the following topics:
...
 
-**Note:** Once your team migrates to Postman v7, the native apps 5.x and 6.x versions and Chrome app 5.x versions will stop syncing and all collaboration features will stop working and all members of the team need to update to their apps to Postman v7 to continue collaborating.
+**Note:** Once your team migrates to Postman v8, the native apps 6.x and 7.x versions will stop syncing and all collaboration features will stop working and all members of the team need to update to their apps to Postman v8 to continue collaborating.
```
Comment (cyrielle-hada) [inline src/pages/docs/administration/migrating-to-v7.md:36]: @sowmyaraj92 can we be more specific on what "collaboration features" are, maybe a redirect to [this doc](https://learning.postman.com/docs/collaborating-in-postman/collaboration-intro/#:~:text=Postman%20allows%20all%20users%20to,within%20your%20team%20and%20workspace.)?

File: src/pages/docs/administration/migrating-to-v7.md [hunk 3]
```diff
@@ -23,24 +27,25 @@ This section describes the following topics:
...
 
 ## How to migrate?
 
-This section illustrates the possible user flows and guides you through the migration process. To use this feature, the entire team must be on Postman v7.
+This section illustrates the possible user flows and guides you through the migration process. To use this feature, the entire team must be on Postman v8.
 
-**Note:** The migration decision is restricted to the team Administrators. If you are a Postman team member and are not on Postman v7, please contact your Administrator to migrate to Postman v7.
+**Note:** The migration decision is restricted to the team Administrators. If you are a Postman team member and are not on Postman v8, please contact your Administrator to migrate to Postman v8.
```
Comment (cyrielle-hada) [inline src/pages/docs/administration/migrating-to-v7.md:42]: It's a minor suggestion here but it seems like we usually use the word **Admin** even in our [Defining roles documentation](https://learning.postman.com/docs/collaborating-in-postman/roles-and-permissions/) and not to **Administrators**.

File: src/pages/docs/administration/migrating-to-v7.md [hunk 4]
```diff
@@ -54,7 +59,7 @@ Once your team has migrated to Postman v7, all users in the team would be notifi
 
 #### Updating app to Postman v7 after migration
 
-You can update your Postman v6 app to Postman v7, via **Settings > Update > Check for Updates**. Postman v7 would be available via auto-updates to all team users after the admin has successfully migrated the team.
+You can update your Postman v6 app to Postman v7, via **Settings > Update > Check for Updates**. Postman v7 would be available via auto-updates to all team users after the admin has successfully migrated the team. In the similar way, you can update Postman v7 app to Postman v8.
```
Comment (cyrielle-hada) [inline src/pages/docs/administration/migrating-to-v7.md:62]: @sowmyaraj92 this is maybe a question to address to Product but if a user is on v6 and want to upgrade, they will first have to go through v7 to then upgrade to v8?

**Review by cyrielle-hada — COMMENTED** [review]
@sowmyaraj92 just a general feedback, could we use **questions** for the headers in a similar way we do for the first two headers?

- Why migrate?
- How to migrate? 

I believe it will help users to go directly to the content they are looking.

Example: instead of download Postman v7 --> Can I roll back to a previous version? Provide the links and explain what it takes to do this (i.e. you need to be an Admin and you need to make sure the whole team rolls back to leverage the collaboration feature).

File: src/pages/docs/administration/migrating-to-v7.md [hunk 5]
```diff
@@ -23,24 +27,25 @@ This section describes the following topics:
 
 ## Why migrate?
 
-If you are a team user, you must migrate to Postman v7 to leverage fine-grained role-based access of Postman’s features. If you stay back on the older version of Postman app, you will not be able to:
+If you are a team user, you must migrate to Postman v8 to leverage fine-grained role-based access of Postman’s features. If you stay back on the older version of Postman, you will not be able to:
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:30]: "leverage fine-grained role-based access" let's change this to fit v8 — @arlemi may have an idea of top things to highlight from release notes?

File: src/pages/docs/administration/migrating-to-v7.md (hunk 2 above)
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:36]: Is this confirmed?

File: src/pages/docs/administration/migrating-to-v7.md (hunk 3 above)
Comment (claireriley) [reply src/pages/docs/administration/migrating-to-v7.md:42]: @sowmyaraj92 let's remove capitalization as well on admin

File: src/pages/docs/administration/migrating-to-v7.md (hunk 4 above)
Comment (claireriley) [reply src/pages/docs/administration/migrating-to-v7.md:62]: I would confirm above as well with product or devrel, I would assume if you opt to upgrade it would go immediately to v8.x or the latest release. 

If that's the case, let's group 7.x in with the "Team on Postman version 5/6" section so it's clear that's in the past and v8 is the now

File: src/pages/docs/administration/migrating-to-v7.md [hunk 6]
```diff
@@ -65,7 +70,7 @@ sudo snap refresh postman
 
 ### Team user on Postman v7
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:71]: I would group this with the previous section if my above comment is true, so the past versions are all grouped in one section. That way it'll quickly move to v8 as the main topic

File: src/pages/docs/administration/migrating-to-v7.md [hunk 7]
```diff
@@ -83,3 +88,24 @@ Please use the following links to download the latest Postman v6 version for you
 sudo snap switch --channel=v6/stable postman
 sudo snap refresh postman
 ```
+
+### Team user on Postman v8
+
+If you are in a team and download Postman v8 when the entire team is on an older version(v7 or below), you will not be able to migrate to Postman v8. Postman displays a message saying you can’t access features on this version of Postman until the team migrates to Postman v8. If you happen to be on Postman v8 and would like to downgrade to Postman v7 to continue working with a team that has not migrated yet, please download the latest Postman v7 app from the links below.
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:94]: "you will not be able to migrate to Postman v8" I would rephrase this more to be like "you will not be able to use Postman v8"

File: src/pages/docs/administration/migrating-to-v7.md [hunk 8]
```diff
@@ -12,9 +12,13 @@ contextual_links:
 
 ---
 
-This section describes the steps/flow to migrate to Postman v7.
+This section describes the steps/flow to migrate to Postman v8.
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:15]: Let's add a TOC

File: src/pages/docs/administration/migrating-to-v7.md [hunk 9]
```diff
@@ -83,3 +88,24 @@ Please use the following links to download the latest Postman v6 version for you
 sudo snap switch --channel=v6/stable postman
 sudo snap refresh postman
 ```
+
+### Team user on Postman v8
+
+If you are in a team and download Postman v8 when the entire team is on an older version(v7 or below), you will not be able to migrate to Postman v8. Postman displays a message saying you can’t access features on this version of Postman until the team migrates to Postman v8. If you happen to be on Postman v8 and would like to downgrade to Postman v7 to continue working with a team that has not migrated yet, please download the latest Postman v7 app from the links below.
+
+## Download latest Postman v7 app
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:96]: I would take out this header and put these as a list in the section right before

File: src/pages/docs/administration/migrating-to-v7.md [hunk 10]
```diff
@@ -83,3 +88,24 @@ Please use the following links to download the latest Postman v6 version for you
 sudo snap switch --channel=v6/stable postman
 sudo snap refresh postman
 ```
+
+### Team user on Postman v8
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:92]: The headings on the previous version are really odd and I think we should update all of them to be more descriptive. I would consider ignoring the existing titles and coming up with an outline separately that makes sense and then using that as a new base. It's "migrating to v8", so my first instinct is the following topics/order:

* "Introducing Postman v8" - intro v8 (the what and why, include the web version)
* "Migrating to Postman v8 as an individual" - that brief note that's at the top rn
* "Migrating to Postman v8 as a team" - note requirement for team as a whole to upgrade simultaneously and that it's recommended
    * "Immediately migrating to Postman v8" - steps for immediate migration 
    * "Delaying migration to Postman v8" - the above is recommended, but you can download postman v7 with these links if you require it

File: src/pages/docs/administration/migrating-to-v7.md [hunk 11]
```diff
@@ -12,9 +12,13 @@ contextual_links:
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v7.md:12]: Let's change the page name and add a redirect

File: src/pages/docs/administration/migrating-to-v7.md (hunk 2 above)
Comment (sowmyaraj92) [reply src/pages/docs/administration/migrating-to-v7.md:36]: @sankalp0o Would love to know your inputs here.

File: src/pages/docs/administration/migrating-to-v7.md (hunk 5 above)
Comment (sowmyaraj92) [reply src/pages/docs/administration/migrating-to-v7.md:30]: @arlemi It would be helpful if you could add your inputs here.

File: src/pages/docs/administration/migrating-to-v7.md (hunk 2 above)
Comment (sowmyaraj92) [reply src/pages/docs/administration/migrating-to-v7.md:36]: @sankalp0o Need a little help here.

Comment (sankalp0o) [reply src/pages/docs/administration/migrating-to-v7.md:36]: Adding the doc link is a good idea. I don't think the update happens like this though. It should be like this - 

Once your team migrates to Postman v7 (or above), the native apps 5.x and 6.x versions and Chrome app 5.x versions will stop syncing and all collaboration features will stop working and all members of the team need to update ~~to~~ their apps to Postman v7 (or above) to continue collaborating.

File: src/pages/docs/administration/migrating-to-v7.md (hunk 5 above)
Comment (arlemi) [reply src/pages/docs/administration/migrating-to-v7.md:30]: Oops missed this. I don't think there are big changes in terms of RBAC between v7 and v8. In terms of general updates you can use the highlights here: 
https://community.postman.com/t/whats-new-in-postman-for-the-web/18203
and this blog post (pretty much the same): 
https://blog.postman.com/introducing-postman-desktop-app/

**Comment by prashantagarwal** [comment]
@sowmyaraj92 I see a common misunderstanding in terms of all the changes here.

- Migration is only required if users are on v6 (wither v6 -> v7 or v6 -> v8). There is no migration required if you are updating the app from v7 to v8. Just a recommendation but rather than specifying v7 or v8, we can everywhere say v7.0.0 and above
- Syncing or collaboration will not be affected in any way for people who are on v7 or v8 in a team. It will only affect users who are on v6 in that team.

Let me know if you want me to comment on specific sections or any other way I could help.

File: src/pages/docs/administration/migrating-to-v8.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+        * [Downloading Postman v6 app](#downloading-postman-v6-app)
+    * [Team on Postman version 5/6](#team-on-postman-version-56)
+
+## Introducing Postman v8.0.0
+
+Postman v8.0.0 gives you a consistent, better API development experience. It has universal search across all of Postman, including the Public and Private API Networks, plus you can work on public workspaces directly through the desktop app. This means that the Postman desktop app and Postman on the [web](https://community.postman.com/t/whats-new-in-postman-for-the-web/18203) are now equally empowering—which one you use is a matter of preference for your workflows.
+
+### Migrating to Postman v7.0.0 or above
+
+Non-team users can move to Postman v7.0.0 or above by updating the app via **Settings > Update > Check for Updates**, without worrying about the steps described in this section.
```
Comment (claireriley) [inline src/pages/docs/administration/migrating-to-v8.md:31]: When a team or user updates, do they update directly to the latest version (v8.x) or do they go to v7 first, then need to upgrade to v8? I assume it's the former, but it's unclear in several places in this doc

Comment (sowmyaraj92) [reply src/pages/docs/administration/migrating-to-v8.md:31]: The user can upgrade directly to v8.

File: src/pages/docs/administration/migrating-to-v7.md (hunk 4 above)
Comment (sowmyaraj92) [reply src/pages/docs/administration/migrating-to-v7.md]: @Cyrielle If a user is on v6, they can directly upgrade to v8 or the latest release.

**Review by claireriley — APPROVED** [review]
LGTM! Made a few small edits.
